keel-web/src/test/kotlin/com/netflix/spinnaker/keel/apidocs/ApiDocTests.kt

```diff
@@ -64,142 +98,131 @@ import strikt.jackson.textValues
     "keel.plugins.ec2.enabled=true",
     "keel.plugins.titus.enabled=true"
   ],
-  webEnvironment = MOCK
+  webEnvironment = NONE
 )
 @AutoConfigureMockMvc
 @EnableAutoConfiguration(exclude = [TaskSchedulingAutoConfiguration::class])
 class ApiDocTests : JUnit5Minutests {
+
   @Autowired
-  lateinit var mvc: MockMvc
+  lateinit var generator: Generator
```

Comment:
This set of tests are sanity checks that the JSON schema renders how we expect. Fundamentally it's testing the same things as before. It no longer needs to hit a controller endpoint to get the schema as we have a generator bean. Also some of the schema structure has changed a bit due to the differences between Open API and JSON schema